contributors/devel/sig-architecture/api-conventions.md

```diff
@@ -1145,11 +1145,11 @@ reduce data volume, load on the system, and noise exposed to users.
 
 ## Naming conventions
 
-* Go field names must be CamelCase. JSON field names must be camelCase. Other
+* Go field names must be Pascal case. JSON field names must be camelCase. Other
```

Comment:
```suggestion
* Go field names must be PascalCase. JSON field names must be camelCase. Other
```
?

Reply:
(UpperCamelCase also works)

Reply:
as someone that was not familiar with Pascal case as a terminology, I agree that this reading is less clear to the average reader (assuming that was me).  I think PascalCase or UpperCamelCase is clearer visual indicator of what was intended.

Reply:
### @derekwaynecarr @sftim @jdumars  : Please review. Have changed it appropriately.